### DISCUSSION TITLE
Inventory issues with favorites

### DISCUSSION THREAD

**Discussion by GettnBetter** [pwillworth/galaxyharvester#21] [General]
I am having trouble figuring out how to use the new favorites features.  I click harvesting, shopping and inventory depending on the desired group.  What I can't figure out is how to move a resource from Harvesting or Inventory (incoming) to just inventory.  I'm finding I have to completely delete the favorite for a resource and add it back manually or it remains in either harvesting or incoming inventory.  The end result is all new resources end up at the bottom of my favorites page and I am constantly scrolling to the bottom of the inventory to see recent inventory after a resource runs out.

I guess the question is, once I have selected harvesting or inventory (incoming), how do I reset the favorite type so it's listed in the same group as the thousands of other inventory items without deleting it completely and manually adding it back to my inventory?  There seems to be two "incoming" options for inventory, neither moves the resource to inventory.

**Comment by pwillworth** [comment]
What you are experiencing is due to all of your resources in your inventory not being in any group.  In the old system when you marked a resource as favorite, it was added to inventory with no group, so it showed up at the top where all  un-grouped things are.  I didn't add a way to add resources from survey/harvesting to a "blank" group.  The idea is you would pick whatever group you want it in from that drop down that defaults to "Incoming".

In the short term, it might clean things up for you if you move all your old resources into some group using the "Move Checked" button on inventory page, then you can add any new ones to that same group and they will all be together.  Long term, we could do some kind of change to make this a little smoother.  Maybe a one time change to move everyone's favorites that are currently un-grouped to the "Incoming" group or something.  I wanted people to have the ability to pop resources directly into a group, and it would be a little awkward to incorporate a way to put it in "no group" as well.  I'll have to think on it a bit though.
